### PR TITLE
[VGNWB-216] Fixed VIAF id not added to equal to field in concept power

### PR DESCRIPTION
**Pull request by prnvshrn**
Fixed the following issues :-
1. VIAF id is added to equal to field in concept power. 
2. Similar to field does not display [] when there are no similar urls.

### CODE REVIEW

File: conceptpower/classes.py [hunk 1]
```diff
@@ -236,8 +236,8 @@ def create(self, user, password, label, pos, conceptlist, description,
             "description": description,
             "type": concepttype,
             "synonymids": synonym_ids,
-            "equals": equal_uris,
-            "similar": similar_uris
+            "equal_to": ",".join(str(equal_uris_element) for equal_uris_element in equal_uris),
+            "similar": ",".join(str(similar_uris_element) for similar_uris_element in similar_uris)
```
Comment (jdamerow) [inline conceptpower/classes.py]: aren't the elements in ```equal_uris``` strings?

File: setup.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 DESCRIPTION = ('A very simple library for querying a Conceptpower REST API.')
 LICENSE = 'GNU GPL 3'
 URL = 'https://github.com/erickpeirson/Conceptpower-API'
-VERSION = '1.9'
+VERSION = '2.0'
```
Comment (jdamerow) [inline setup.py]: let's wait with increasing the version number until we're actually releasing a version

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: conceptpower/classes.py (hunk 1 above)
Comment (prnvshrn) [reply conceptpower/classes.py]: I have removed the string typecast in both <b>equal_to</b> and <b>similar</b> fields

File: setup.py (hunk 2 above)
Comment (prnvshrn) [reply setup.py]: Reverted back the version to <b>1.9</b> from <b>2.0</b>

**Review by jdamerow — APPROVED** [review]

**Pull request merged**